server/src/db/migrations/1_initial-migrations.sql

```diff
@@ -1,6 +1,15 @@
-CREATE TABLE student (
-  student_id SERIAL PRIMARY KEY,
+CREATE EXTENSION "uuid-ossp";
+
+CREATE TABLE "user" (
+  user_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
   first_name TEXT NOT NULL,
   last_name TEXT NOT NULL,
-  birth_year INT NOT NULL
+  roles TEXT[] NOT NULL CHECK (roles <@ '{"para", "case_manager", "admin"}')
```

Comment:
I think having only one option is better. The current three - admin, case_manager and para are hierarchical in that being an admin can allow you to act as everything, being a CM, you should be able to acts as a para, and the para is the leaf. We can assign folks the highest level role.

Here are some of the responsibilities of the different roles:
admin: misc like creating other roles :)
case_manager: create IEP and tasks, assign paras and schedule
para: collect attempt information for a particular task

Reply:
I was originally thinking in terms of RBAC when modeling this. I think it's definitely possible that we could introduce new roles in the future that aren't hierarchical, but yeah it's probably simpler to limit to a single role for now.

Reply:
changed in 0b77c5c